staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -105,13 +110,14 @@ func NewNamedReflector(name string, lw ListerWatcher, expectedType interface{},
 	r := &Reflector{
 		name: name,
 		// we need this to be unique per process (some names are still the same) but obvious who it belongs to
-		metrics:       newReflectorMetrics(makeValidPrometheusMetricLabel(fmt.Sprintf("reflector_"+name+"_%d", reflectorSuffix))),
-		listerWatcher: lw,
-		store:         store,
-		expectedType:  reflect.TypeOf(expectedType),
-		period:        time.Second,
-		resyncPeriod:  resyncPeriod,
-		clock:         &clock.RealClock{},
+		metrics:            newReflectorMetrics(makeValidPrometheusMetricLabel(fmt.Sprintf("reflector_"+name+"_%d", reflectorSuffix))),
```

Comment:
nit: I realize this line was here already but since we are here and since we are also using Sprintf here, would you mind passing name in as an argument to the fmt call?

Reply:
sure, np.